code/modules/mapfluff/ruins/spaceruin_code/infested_frigate.dm

```diff
@@ -0,0 +1,2 @@
+/area/ruin/space/infested_frigate
+    name = "SYN-C Brutus"
```

Comment:
you can put this in `code/game/area/areas/ruins/space.dm`

Reply:
if i did this do i hit resolve conversation

Reply:
yes

Reply:
but you need to delete it from here dont add it twice
and then delete this file